exporter/metric/prometheus/prometheus.go

```diff
@@ -39,11 +35,11 @@ type Exporter struct {
 	registerer prometheus.Registerer
 	gatherer   prometheus.Gatherer
 
-	counters           counters
-	gauges             gauges
-	histograms         histograms
-	summaries          summaries
-	measureAggregation MeasureAggregation
+	snapshot export.CheckpointSet
+	OnError  func(error)
+
+	DefaultHistogramBuckets  []float64
+	DefaultSummaryObjectives map[float64]float64
```

Comment:
should this be exported?

Reply:
I think the name "Objectives" is borrowed from Prometheus, where it refers to the desired error ratios. Let's name this "DefaultSummaryQuantiles" as long as we're not using the objectives. It can be a []float64 since it's just a list of quantile values, for now.

Reply:
No they should't be exported, I have unexported these fields and changed the `DefaultSummaryObjectives`.

Thanks!